components/employmentOutlook/FavorabilityDialog.js

```diff
@@ -0,0 +1,152 @@
+import { fade } from '@material-ui/core/styles/colorManipulator';
+import { makeStyles } from '@material-ui/styles';
+import Button from '@material-ui/core/Button';
+import Dialog from '@material-ui/core/Dialog';
+import Divider from '@material-ui/core/Divider';
+import NextLink from 'next/link';
+import PropTypes from 'prop-types';
+import React from 'react';
+import Typography from '@material-ui/core/Typography';
+
+import { DialogContent, DialogTitle, DialogActions } from '../DialogComponents';
+
+const useStyles = makeStyles(theme => ({
+  title: {
+    textAlign: 'center',
+    verticalAlign: 'center',
+    paddingTop: theme.spacing(6),
+    paddingBottom: theme.spacing(4),
+  },
+  emphasis: {
+    fontWeight: 'bold',
+    textDecorationLine: 'underline',
+  },
+  summary: {
+    marginTop: theme.spacing(4),
+    marginBottom: theme.spacing(4),
+  },
+  description: {
+    marginTop: theme.spacing(4),
+    marginBottom: theme.spacing(4),
+  },
+  button: {
+    border: 'none',
+    padding: 0,
+    textDecoration: 'underline',
+  },
+  footer: {
+    justifyContent: 'center',
+    padding: theme.spacing(2, 0, 2, 0),
+  },
+}));
+
+const FAVORABILITY_TYPE = [
+  {
+    value: 'Very Unfavorable',
+    color: '#f96861',
+    growth: 'Declining',
```

Comment:
Growth and size are not static -- their values may differ for a given overall favorability. Please use real data.

Reply:
done